### PR TITLE
bugfixs to get --log-level to work again

### PR DESCRIPTION
**Pull request by clintwood**
Fixes:

- Change command line `--loglevel`  to `--log-level` to satisfy `logLevel: logLevel` option in `bin/index.js`.
- Bugfix: Ensure correct `options.logLevel` option is used in `lib/index.js, lib/pouchdb-server-lite.js`
- Bugfix: Correct `level.toUpperCase()` (instead of `level.toUpperCaseString()`) in `lib/logger.js`  
- Enhance: Pass existing `logger` instance like `util.request(logger)` in  `lib/index.js, lib/pouchdb-server-lite.js` instead of creating a default logger without the configured `--log-level` in `lib/logger.js`.

### CODE REVIEW

File: bin/index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ program
   .version(require('../package.json').version)
   .option('-p, --port [port]', 'The port to run local-npm on', 5080)
   .option('-pp, --pouch-port [port]', 'The port to run the pouch db server on', 16984)
-  .option('-l, --loglevel [level]', 'The level to log information to the console from local-npm', 'error')
+  .option('-l, --log-level [level]', 'The level to log information to the console from local-npm', 'error')
```
Comment (gabrielcsapo) [inline bin/index.js]: 👍

File: lib/logger.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ function Logger(level) {
   }
 }
 Logger.getLevel = function getLevel(level) {
-  if(level) level = level.toLowerCaseString();
```
Comment (gabrielcsapo) [inline lib/logger.js]: good catch

File: lib/util.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
-const logger = new (require('./logger'))();
 const responseTime = require('response-time');
 
-module.exports.request = (req, res, next) => {
+module.exports.request = (logger) =>
```
Comment (gabrielcsapo) [inline lib/util.js]: 👍 I like this a lot more then what I had before

**Comment by gabrielcsapo** [comment]
@clintwood awesome fixes, thank you!

**Comment by gabrielcsapo** [comment]
published changes under `2.2.1` https://github.com/local-npm/local-npm/releases/tag/2.2.1

File: lib/util.js (hunk 3 above)
Comment (clintwood) [inline lib/util.js]: Awesome!

**Pull request merged**